tests/playground.html

```diff
@@ -228,8 +228,8 @@
           blockToDrag.onMouseUp_(mouseUpEvent);
 
           setTimeout(fakeDragWrapper(), 100);
-        }, 30);
-    }, 30);
+        }, 100);
```

Comment:
Yes, it is.  I've been using this for testing, but if a fake drag took too long I could get mouseUp before mouseDown finished.  This slows it down for better ordering guarantees.